pootle/static/js/shared/components/KeyboardHelpDialog.js

```diff
@@ -11,14 +11,8 @@ import { showModal } from './Modal';
 
 import './KeyboardHelpDialog.css';
 
-let keyboardHelpOpen = false;
 
 export function showKeyboardHelpDialog() {
-  if (keyboardHelpOpen) {
-    return;
```

Comment:
I used this `keyboardHelpOpen` logic to avoid opening multiple layers of keyboard help popup on top of each other. With this code removed, the issue will be back (I checked).